addons/evm/src/commands/actions/check_confirmations.rs

```diff
@@ -25,28 +25,32 @@ lazy_static! {
             implements_background_task_capability: true,
             inputs: [
                 tx_hash: {
-                  documentation: "The transaction hash to check.",
-                  typing: Type::buffer(),
-                  optional: false,
-                  interpolable: true
+                    documentation: "The transaction hash to check.",
+                    typing: Type::buffer(),
+                    optional: false,
+                    tainting: true,
+                    internal: false
                 },
                 rpc_api_url: {
-                  documentation: "The URL of the EVM API used to poll for the transaction's inclusion in a block.",
-                  typing: Type::string(),
-                  optional: false,
-                  interpolable: true
+                    documentation: "The URL of the EVM API used to poll for the transaction's inclusion in a block.",
+                    typing: Type::string(),
+                    optional: false,
+                    tainting: true,
```

Comment:
It's interesting that the api url isn't tainting for call_contract, but it is for this - is that on purpose?

Reply:
Addressed with latest